src/mam/mod_mam_cache_user.erl

```diff
@@ -13,78 +11,58 @@
 %%%-------------------------------------------------------------------
 -module(mod_mam_cache_user).
 
--behaviour(gen_mod).
 -behaviour(mongoose_module_metrics).
 
 %% gen_mod handlers
--export([start/2, stop/1, supported_features/0]).
+-export([start/2]).
+-export([deps/2]).
+-export([stop/1]).
+-export([supported_features/0]).
```

Comment:
I have to say I don't see much benefit in this verbose convention. I know it's trying to be smart by reducing diff size, but actually these four lines could be just one. To quote one of our colleagues, "It's a man doing a machine's job" - that one was about the comma-first convention, but I think it's relevant here as well - I don't think we should make source code more cluttered to make the future diffs shorter - cleaner code wins over cleaner version history for me.

Reply:
Not any preference here, can easily change 👍🏽